redhat-developer/app-services-cli
# feat(kafka create): add billing model logic

This PR adds `--billing-model` flag to `rhoas kafka create` command.
The CLI should be able to create instances with standard, marketplace and trial quotas. 

### Verification Steps

1. For testing against marketplace  (use _kafka_service account).

`rhoas kafka create --name test-instance` should throw error saying multiple quota types are available and user needs to specify billing model.
`rhoas kafka create --name test-instance --billing-model standard` should create a standard instance
`rhoas kafka create --name test-instance --billing-model standard` should create a marketplace instance

`rhoas kafka create` should start the creation in interactive mode and prompt for billing model.

User should be able to pick marketplace provider and market place account if multiple options are available, else the only account gets selected by default.

2. When only trial instances are available (use _kafka_devexp account)

Specifying the billing model should fail with message "only trial instances available".

`rhoas kafka create --name test-instance` should create a trial instance.

Interactive mode should not prompt billing model.

### Type of change
